openmc/surface.py

```diff
@@ -949,6 +949,152 @@ def __init__(self, surface_id=None, boundary_type='transmission',
         self._type = 'z-cone'
 
 
+class Quadric(Surface):
+    """A sphere of the form :math:`Ax^2 + By^2 + Cz^2 + Dxy + Eyz + Fxz + Gx + Hy +
+    Jz + K`.
+
+    Parameters
+    ----------
+    surface_id : int
+        Unique identifier for the surface. If not specified, an identifier will
+        automatically be assigned.
+    boundary_type : {'transmission, 'vacuum', 'reflective', 'periodic'}
+        Boundary condition that defines the behavior for particles hitting the
+        surface. Defaults to transmissive boundary condition where particles
+        freely pass through the surface.
+    a, b, c, d, e, f, g, h, j, k : float
```

Comment:
Ok, yes, I see that is the case in the `Surface.create_xml_subelement(...)` routine.